src/main/java/core/basesyntax/service/impl/CsvWriterImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.CsvWriter;
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class CsvWriterImpl implements CsvWriter {
+    @Override
+    public void write(String fileName, String string) {
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(fileName))) {
+            writer.write(string);
+            writer.flush();
```

Comment:
you don't need to call flush explicitly. If I'm not wrong it'll be called automatically when the writer is going to be closed